docs/notebooks/flax_basics.ipynb

```diff
@@ -0,0 +1,911 @@
+{
```

Comment:
<p>Looks like there's more to add for the state section. Maybe it should actually be a separate doc because there's a lot to talk about here, such as how you use an optimizer when you have both parameters as well as non-trainable variables. LMK if you want to chat more about this, or when you have a draft that you'd like to have reviewed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/flax/pull/542/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>Let's chat indeed as I'm not sure about how we should tackle this. Also the @andsteing comment above.</p>